project-1/gspread_code.py

```diff
@@ -0,0 +1,86 @@
+import os
+import logging 
+import gspread
+import pandas as pd
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+
+logging.basicConfig(level=logging.DEBUG, 
+                    format="%(levelname)s (%(asctime)s) : %(message)s (%(lineno)d)")
+ 
+current_working_directory = os.getcwd()
+#project_json is the credential file for the service account created
+project_json = "spartan-figure-402211-2241d0349aab.json"
```

Comment:
This name for your file is too specific you can move it to your .env and call it CREDENTIAL_FILE="....aab.json"

Reply:
Alright sir, do you mean i should just move the file to the project directory and rename the file to something more descriptive??

Reply:
move it to .env file:

`PROJECT_JSON = zxxxxxx.json`

Reply:
Check out how to use `.env` in python with python-dotenv library